package.json

```diff
@@ -38,19 +38,21 @@
     "@apidevtools/json-schema-ref-parser": "^11.1.0",
     "@apidevtools/swagger-parser": "^10.1.0",
     "@asyncapi/parser": "^3.0.14",
```

Comment:
Hello @jonaslagoni Could you please also bump version of parser to the newest? They fixed one important vulnerability. Thank in advice!